pkg/devfile/adapters/common/command.go

```diff
@@ -109,7 +117,9 @@ func validateCommandsForGroup(data data.DevfileData, groupType common.DevfileCom
 
 	if len(commands) > 1 {
 		for _, command := range commands {
-			if command.Exec.Group.IsDefault {
+			if command.Composite != nil && command.Composite.Group.IsDefault {
```

Comment:
we're validating command defaults here by group kind, so i think even this if and else if can be combined for a single condition..

Reply:
Can we add some comp cmd unit test cases for this func?

Reply:
Done